util/binwarp/binwarp.lisp

```diff
@@ -0,0 +1,105 @@
+;;;; binwarp.lisp
+
+(in-package #:binwarp)
+
+(declaim (special *binwarp-mode-p* *binwarp-area* *binwarp-history*
+                  *reinitiate-ptr* *init-ptr-position*
+                  *preserve-history*))
+(defvar *binwarp-mode-p*     nil "Whether you are in the binwarp mode.")
+(defvar *binwarp-area*       nil "The current screen area that binwarp is restricted by.")
+(defvar *binwarp-history*    nil "The previous binwarp areas.")
```

Comment:
If this is a list, consider hinting so by using `()`, which is a synonym of `nil`.

```suggestion
(defvar *binwarp-history*    () "The previous binwarp areas.")
```

Reply:
Shouldn't it be a literal list (`'()`) ? Or is empty lists quoting optional?

Reply:
In CL we don't need to quote `()`. Only in Scheme. Some people recommend doing so anyway and although I disagree that is the style that you'll find in the StumpWM codebase.